qa/vagrant/src/test/resources/packaging/scripts/plugins.bash

```diff
@@ -36,6 +36,8 @@ install_plugin() {
 
     assert_file_exist "$ESPLUGINS/$name"
     assert_file_exist "$ESPLUGINS/$name/plugin-descriptor.properties"
+    #check we did not accidentially create a log file as root as /usr/share/elasticsearch
+    assert_file_not_exist "/usr/share/elasticsearch/logs"
```

Comment:
Not sure what this means. I would actually expect an output to stdout now even more than before?